src/lwip/k32w/lwipopts.h

```diff
@@ -94,7 +94,7 @@
 #define MEMP_SEPARATE_POOLS (1)
 #define LWIP_PBUF_FROM_CUSTOM_POOLS (0)
 #define MEMP_USE_CUSTOM_POOLS (0)
-#define PBUF_POOL_SIZE (8)
+#define PBUF_POOL_SIZE (6)
```

Comment:
Does anybody know the absolute minimum that would be required to support Matter?

Reply:
I'm not sure about it, but lots of platform is using 6